From 772e69f09d90f5d8070bb11001410ac3dd23ae15 Mon Sep 17 00:00:00 2001
From: Sebastien MICHEL <sebastien.michel@intel.com>
Date: Fri, 14 Jun 2013 09:53:11 +0200
Subject: [PORT FROM R42TW] Optimization via CFLAG options

BZ: 129203

Add -fno-pic and -fno-pie to get faster execution needed to get RLBench
target

Change-Id: I36e629537b5970504c60891fcb5efc0f522351cf
Orig-Change-Id: Id70c120e347918e12421f068cf9997d8e34e8eb6
Category: device enablement
Domain: AOSP.Optimization-Bionic
Origin: internal
Upstream-Candidate: no
Signed-off-by: Sebastien MICHEL <sebastien.michel@intel.com>
---
 dist/Android.mk |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/dist/Android.mk b/dist/Android.mk
index fa76eae..e986a56 100644
--- a/dist/Android.mk
+++ b/dist/Android.mk
@@ -47,6 +47,12 @@ ifneq ($(TARGET_ARCH),arm)
 LOCAL_LDLIBS += -lpthread -ldl
 endif
 
+ifeq ($(TARGET_ARCH),x86)
+common_sqlite_flags += \
+    -fno-pic \
+    -fno-pie
+endif
+
 LOCAL_CFLAGS += $(android_common_sqlite_flags) $(common_sqlite_flags) -DUSE_PREAD64 -Dfdatasync=fdatasync \
 				-DHAVE_MALLOC_USABLE_SIZE
 
-- 
1.7.4.1

